airflow/bin/cli.py

```diff
@@ -1228,13 +1228,16 @@ def create_user(args):
         password_confirmation = getpass.getpass('Repeat for confirmation:')
         if password != password_confirmation:
             raise SystemExit('Passwords did not match!')
-
-    user = appbuilder.sm.add_user(args.username, args.firstname, args.lastname,
-                                  args.email, role, password)
-    if user:
-        print('{} user {} created.'.format(args.role, args.username))
+    user = appbuilder.sm.find_user(args.username, args.email)
+    if not user:
+        user = appbuilder.sm.add_user(args.username, args.firstname, args.lastname,
+                                      args.email, role, password)
+        if user:
+            print('{} user {} created.'.format(args.role, args.username))
+        else:
+            raise SystemExit('Failed to create user.')
     else:
-        raise SystemExit('Failed to create user.')
```

Comment:
@Fokko I'd like your thoughts on how best to handle this. Currently since the airflow-init creates a new user. If the user already exists the airflow deployment fails. However, I want to be certain we're not changing the functionality in a detrimental way.

Reply:
Hi @dimberman, I see the issue. If you don't recreate the database, the user will be available. Changing the SystemExit is not a nice way to solve this. I would just add `|| true` to the create user command to ignore the exit code: `airflow create_user -u airflow -l airflow -f jon -e airflow@apache.org -r Admin -p airflow || true`. What do you think?

Reply:
@Fokko So I guess the big question is: Should the system return an error if I attempt to create a user that already exists. That said, I think it's totally valid to say that the answer to that question is outside of the scope of this PR, so I'll just do the `|| true`.

Reply:
Yes, when you invoke the cli with the action to create an user, and the user already exists. I think `exit code != 0` is the right thing to do.